### DISCUSSION TITLE
Group automatisations

### DISCUSSION THREAD

**Discussion by rattatosk** [home-assistant/frontend#11497] [Automations / scripts / scenes] [answered]
Hi
I have a lot of automatisations set up in house, but now in the  HA UI it is a big long list in the Automations list.
To have some overview, I prefix every automatisation with the zone.
e.g. Living_Light_Sensor_Auto, Garden_Light_Clock, etc etc. But this is still rather disorganised.

It would help a lot if we could assign group names (or tags) to group the automatisations and also have them presented in the UI by group. e.g by Group in the same list (idented under the gorup name) or in different tabs.

Thanks in advance :)

**Comment by frenck** [comment, marked as answer]
> It would help a lot if we could assign group names (or tags) to group the automatisations and also have them presented in the UI by group. e.g by Group in the same list (idented under the gorup name) or in different tabs.
 
They can be filtered by areas and even by devices?

![image](https://user-images.githubusercontent.com/195327/151832720-413795d8-5530-42c2-9c59-5bc4f14f90e0.png)

**Reply by rattatosk** [reply]
Well duh... I didn't see that.. Thanks !

It might improve to move the settings window to edit these parameters for the automation, on the automation screen itself.
If I'm not mistaken, I can only reach this through the info icon on the main list, and then clicking the cogwheel. Which doesn't feel as intuitive.

Thanks for the help !
